services/wallet/cryptocompare.go

```diff
@@ -127,3 +145,57 @@ func fetchTokenMarketValues(symbols []string, currency string) (map[string]Marke
 	return item, nil
 
 }
+
+func fetchHourlyMarketValues(symbol string, currency string, limit int, aggregate int) ([]TokenHistoricalPairs, error) {
+	item := []TokenHistoricalPairs{}
+	httpClient := http.Client{Timeout: time.Minute}
+
+	url := fmt.Sprintf("https://min-api.cryptocompare.com/data/v2/histohour?fsym=%s&tsym=%s&aggregate=%d&limit=%d&extraParams=Status.im", symbol, currency, aggregate, limit)
+	resp, err := httpClient.Get(url)
```

Comment:
Probably makes sense to add a client timeout here

Reply:
Hey on line 151, its already defined? 	
`httpClient := http.Client{Timeout: time.Minute}`